### PR TITLE
User/t kazhua/galleryapp sratch

### PR DESCRIPTION
**Pull request by t-kazhua**
Skeletal Detector Skill View with default sample output

### CODE REVIEW

**Comment by LPBourret** [comment]
For image files you added, they need to be published under a creative common license or approved by the owner. Are they compliant?

File: samples/GalleryApp/GalleryApp.csproj [hunk 1]
```diff
@@ -115,9 +115,22 @@
...
+    <Compile Include="..\Common\cs\ImageFileFrameSource.cs">
+      <Link>FrameSource\ImageFileFrameSource.cs</Link>
+    </Compile>
+    <Compile Include="..\Common\cs\MediaPlayerFrameSource.cs">
+      <Link>FrameSource\MediaPlayerFrameSource.cs</Link>
+    </Compile>
     <Compile Include="App.xaml.cs">
       <DependentUpon>App.xaml</DependentUpon>
     </Compile>
+    <Compile Include="BodyRenderer.cs" />
```
Comment (LPBourret) [inline samples/GalleryApp/GalleryApp.csproj]: >BodyRenderer.cs [](start = 22, length = 15)

can this be instead a reference to the existing BodyRender.cs?

File: samples/GalleryApp/MainPage.xaml [hunk 2]
```diff
@@ -39,7 +39,7 @@
                                Padding="10, 60, 0, 0"
                                TextWrapping="Wrap"/>
                 </Grid>
-                <Button Height="400" Width="400" Opacity="0" Click="NavigateToSkillPage"/>
+                <Button Height="400" Width="400" Opacity="0.5" Click="NavigateToSkillPage" Style="{StaticResource ButtonRevealStyle}"/>
```
Comment (LPBourret) [inline samples/GalleryApp/MainPage.xaml]: >ButtonRevealStyle [](start = 114, length = 17)

where is this defined?

File: samples/GalleryApp/SkeletalDetectorPage.xaml [hunk 3]
```diff
@@ -50,30 +50,65 @@
...
+                <Grid Name="UIVideoFeed">
+                    <!-- Video display -->
+                    <Image Name="UIImageViewer" 
+                       Canvas.ZIndex="-1"  
+                       VerticalAlignment="Center"
+                       HorizontalAlignment="Center"
+                       Stretch="Uniform" 
+                       SizeChanged="UIImageViewer_SizeChanged"/>
+
+
```
Comment (LPBourret) [inline samples/GalleryApp/SkeletalDetectorPage.xaml]: nit: remove 1 empty line

File: samples/GalleryApp/SkeletalDetectorPage.xaml.cs [hunk 4]
```diff
@@ -34,5 +63,325 @@ private void Back_Click(object sender, RoutedEventArgs e)
...
+
+            // Run skill on a default image
+            string path = Directory.GetCurrentDirectory();
+            StorageFile file = await StorageFile.GetFileFromPathAsync(path + "\\SampleImages\\People2.jpg");
+            await ConfigureFrameSourceAsync(file);
+            RunSkill_Execution();
+        }
+
+
+
```
Comment (LPBourret) [inline samples/GalleryApp/SkeletalDetectorPage.xaml.cs]: nit: remove line space

**Comment by LPBourret** [comment]
> // WARNING: Please don't edit this file. It was generated by C++/WinRT v1.0.190111.3

Don't add any of the files under inc/ they are generated at compile time

---
Refers to: samples/ObjectDetector/cpp/ObjectDetectorSample_Desktop/inc/winrt/impl/Microsoft.AI.Skills.SkillInterfacePreview.0.h:1 in c398780. [](commit_id = c39878004848bb8c28860f59d9cc8720d26807cf, deletion_comment = False)

File: samples/GalleryApp/MainPage.xaml (hunk 2 above)
Comment (t-kazhua) [reply samples/GalleryApp/MainPage.xaml]: It's a keyword from the UWP style documentation: [https://docs.microsoft.com/en-us/windows/uwp/design/style/reveal](https://docs.microsoft.com/en-us/windows/uwp/design/style/reveal)


---
In reply to: [296978865](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/35#discussion_r296978865) [](ancestors = 296978865)

**Comment by t-kazhua** [comment]
I downloaded from the website, so I don't think they are compliant. I will remove them. 

---
In reply to: [505244854](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/35#issuecomment-505244854) [](ancestors = 505244854)

File: samples/GalleryApp/GalleryApp.csproj (hunk 1 above)
Comment (LPBourret) [reply samples/GalleryApp/GalleryApp.csproj]: yes

---
In reply to: [297270849](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/35#discussion_r297270849) [](ancestors = 297270849,296978554)

File: samples/GalleryApp/GalleryApp.csproj [hunk 5]
```diff
@@ -12,7 +12,7 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
     <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.17763.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.16299.0</TargetPlatformMinVersion>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
```
Comment (marianan) [inline samples/GalleryApp/GalleryApp.csproj]: >17763 [](start = 35, length = 5)

we should add the platform requirements to the documentation.

File: samples/GalleryApp/GalleryApp.csproj [hunk 6]
```diff
@@ -133,7 +148,6 @@
   </ItemGroup>
   <ItemGroup>
     <Content Include="Assets\GitHub-Mark-32px.png" />
```
Comment (marianan) [inline samples/GalleryApp/GalleryApp.csproj]: >GitHub-Mark-32px.png [](start = 29, length = 20)

Could you please check with Varsha if there are any restrictions with using the GitHub logo in the gallery app?

Comment (t-kazhua) [reply samples/GalleryApp/GalleryApp.csproj]: Will do.

**Pull request merged**